### ISSUE TITLE
Documentation and spell/grammar checking

### ISSUE THREAD

**Issue by AlekSi** [AlekSi/reform#8]

**Comment by rumyantseva** [comment]
If you work with dates from reform, you have to add in your model method `afterFind` which  will covert time zones etc. Am I right? 

Maybe it is possible to add some helpful tips for different data type to understand the best way?

**Comment by AlekSi** [comment]
> If you work with dates (like time.Time) from reform, you have to add in your model method afterFind which will convert time zones etc. Am I right?

No, if your database driver is doing a sane thing. But yeah, this need to be documented.

**Comment by AlekSi** [comment]
`ErrNoRows` can be returned from some methods and is never returned from others. General rule probably can be better documented.

**Comment by sebastianmacias** [comment]
How can I use multiple conditions?

`// Find records by IDs.
persons, err := DB.FindAllFrom(PersonTable, "id", 1, 2)
if err != nil {
    log.Fatal(err)
}
for _, p := range persons {
    fmt.Println(p)
}`

Allows me to select by id, let's say I want to select by 'id' and 'name' or any other field.

What would my code look like?

**Comment by AlekSi** [comment]
Various Find methods accept a single column only. More complex conditions are possible with Select methods:
```go
persons, err := DB.SelectAllFrom(PersonTable, "WHERE id = ? AND name = ?", id, name)
```
